sdk/eventhub/ci.yml

```diff
@@ -41,6 +41,20 @@ stages:
     Artifacts:
     - name: Azure.Messaging.EventHubs
       safeName: AzureMessagingEventHubs
-    - name: Azure.Messaging.EventHubs.CheckpointStore.Blob
-      safeName: AzureMessagingEventHubsCheckpointStoreBlob
-      
\ No newline at end of file
+    - name: Azure.Messaging.EventHubs.CheckpointStore.Blobs
+      safeName: AzureMessagingEventHubsCheckpointStoreBlobs
+    - name: Microsoft.Azure.EventHubs
+      safeName: MicrosoftAzureEventHubs
+      options:
+        skipTagRepository: true
```

Comment:
Why are we skipping these steps? Is there a reason we need to publish doc but not packages from this pipeline? We should really ever only publish docs when we publish packages.

Reply:
@mitchdenny  did not have Track one packages added at all to the Artifacts object and am not sure why but he suggested addigt that option for those that shouldn't go to the release step. I didn't want to include anything he didn't already have included.

Reply:
If this is just about the track-1 packages that we want to produce an initial set of docs for I'd suggest using another pipeline (perhaps the client or mgmt) that builds them instead of introducing a bunch of conditions in the pipelines for package we don't want to ship.